plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/status-filter/frontend.ts

```diff
@@ -8,7 +8,7 @@ import { getContext, getElement, store } from '@woocommerce/interactivity';
  */
 import type { ProductFiltersContext } from '../../frontend';
 
-const filterStockStatusKey = 'filter_stock_status';
+const filterStatusKey = 'filter_status';
```

Comment:
So you think that we should drop the old key, right? Just want to note that we will need to update the Product Collection to make the new key work for the case that merchants add the filters and PC to the page (not the template) and when filter blocks are placed inside PC.

Reply:
Currently the experience of having the new filters inside the PC is very broken (useable). So we could fix this later.